### ISSUE TITLE
The font in the tabs are not the same as the font in settings

### ISSUE THREAD

**Issue by egornagornov** [arvi/Agila-Theme#24]
![imageedit_6_2649547137](https://cloud.githubusercontent.com/assets/5019877/21952421/8571fc82-da4e-11e6-910b-7c1922147437.gif)

**Comment by arvi** [comment]
@Exegor, yep, you'd need to add this just below the theme you are using and indicate your preferred font.

https://gist.github.com/arvi/af18cbda263f35ba56dc34231c33ca50

Maybe, I'll create a font face add-on package some time this month when I have more free time to do so :smile:
